avogadro/qtgui/CMakeLists.txt

```diff
@@ -1,3 +1,15 @@
+find_package(Eigen3 REQUIRED)
+# Add as "system headers" to avoid warnings generated by them with
+# compilers that support that notion.
+include_directories(SYSTEM ${EIGEN3_INCLUDE_DIR})
+
+cmake_minimum_required(VERSION 3.5)
```

Comment:
None of this makes sense and is not the direction we want to move the CMake code in this project.

Reply:
mistake it remove